checklists/appsvc_checklist.json

```diff
@@ -0,0 +1,304 @@
+      "guid": "b123071a-5416-4415-a33e-a3ad2c2de732",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/networking/app-gateway-with-service-endpoints"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Avoid for WAF to be bypassed",
+      "description": "Make sure the WAF cannot be bypassed by locking down access to only the WAF.  Use a combination of Access Restrictions, Service Endpoints and Private Endpoints.",
+      "guid": "165c3acb-ef4a-4be1-b8d3-9fda47768314",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/networking-features#access-restrictions"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Set minimum TLS policy to 1.2",
+      "description": "Set minimum TLS policy to 1.2 in App Service configuration.",
+      "guid": "c115775c-2ea5-45b4-9ad4-8408ee72734b",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/configure-ssl-bindings#enforce-tls-versions"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Use HTTPS only",
+      "description": "Configure App Service to use HTTPS only.  This causes App Service to redirect from HTTP to HTTPS.  Strongly consider the use of HSTS in your code or from your WAF.",
+      "guid": "475ba18f-dbf5-490c-b65d-e8e03f9bcbd4",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/configure-ssl-bindings#enforce-https"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Wildcards must not be used for CORS",
+      "description": "Do not use wildcards in your CORS configuration, as this allows all origins to access the service (thereby defeating the purpose of CORS). Specifically only allow the origins that you expect to be able to access the service.",
+      "guid": "68266abc-a264-4f9a-89ae-d9c55d04c2c3",
+      "severity": "High",
+      "link": "https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-rest-api"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
+      "text": "Use an established Identity Provider for authentication",
+      "description": "For authenticated web application, use a well established Identity Provider like Azure AD or Azure AD B2C.  Leverage the application framework of your choice to integrate with this provider or use the App Service Authentication / Authorization feature.",
+      "guid": "919ca0b2-c121-459e-814b-933df574eccc",
+      "severity": "Medium",
+      "link": "https://docs.microsoft.com/azure/app-service/overview-authentication-authorization"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Network Security",
```

Comment:
idem

Reply:
Why not? This disables the remote debugging port at the network level.